mods/cnc/maps/shellmap/map.yaml

```diff
@@ -994,18 +1025,102 @@ Rules:
 			Effect: Desaturated
 		LuaScript:
 			Scripts: shellmap.lua
+	^Vehicle:
+		GivesBounty:
+			Percentage: 0
+		GainsExperience:
+			Upgrades:
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+	^Tank:
+		GivesBounty:
+			Percentage: 0
+		GainsExperience:
+			Upgrades:
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+	^Infantry:
+		GivesBounty:
+			Percentage: 0
+		GainsExperience:
+			Upgrades:
+		DeathSounds@NORMAL:
+			VolumeMultiplier: 0.1
+		DeathSounds@BURNED:
+			VolumeMultiplier: 0.1
+		DeathSounds@ZAPPED:
+			VolumeMultiplier: 0.1
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+	^Ship:
+		GivesBounty:
+			Percentage: 0
+		GainsExperience:
+			Upgrades:
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+	^Plane:
+		GivesBounty:
+			Percentage: 0
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+	^Building:
+		GivesBounty:
+			Percentage: 0
+		GainsStatUpgrades:
+		InvulnerabilityUpgrade@UNKILLABLE:
+			UpgradeTypes: unkillable
+			UpgradeMinEnabledLevel: 1
+		UpgradeManager:
+	^CivInfantry:
+		-ActorLostNotification:
 	LST:
 		Mobile:
-			Speed: 42
+			Speed: 82
 	BOAT:
 		Mobile:
 			Speed: 42
+	MTNK:
+		Buildable:
+			Prerequisites: anyhq
+	LTNK:
+		Buildable:
+			-Prerequisites:
```

Comment:
I'm not sure, but do we need this?
I mean, you say what the player builds with lua.